kuksa_apps/kanto/edge_client.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+import signal
+import sys
+
+import paho.mqtt.client as mqtt
+from ditto.client import Client
+from ditto.model.feature import Feature
+from ditto.protocol.things.commands import Command
+from kuksa_viss_client import KuksaClientThread
+
+from edge_device_info import EdgeDeviceInfo
+from utils import process_tree, process_signal
+
+FEATURE_ID_VSS = "VSS"
+
+EDGE_CLOUD_CONNECTOR_TOPIC_DEV_INFO = "edge/thing/response"
+EDGE_CLOUD_CONNECTOR_TOPIC_DEV_INFO_REQUEST = "edge/thing/request"
+
+
+class EdgeClient:
+    def __init__(self):
+        self.mqtt_client = None
+        cfg = {"ip": "127.0.0.1"}
```

Comment:
The configuration of the application should be refactored to use the kuksa-aligned approach for apps - i.e. configuration .ini file expected in a certain location backed up with default values if missing.